src/test/java/org/reaktivity/nukleus/oauth/internal/control/ControlIT.java

```diff
@@ -46,11 +51,23 @@
         .responseBufferCapacity(4096)
         .counterValuesBufferCapacity(4096)
         .nukleus("oauth"::equals)
-        .configure("oauth.keys", "keys/keys.jwk");
+        .configure(KEYS.name(), "keys/keys.jwk");
 
     @Rule
     public final TestRule chain = outerRule(k3po).around(timeout).around(reaktor);
 
+    @Test
+    public void shouldMatchKeysConfigName()
+    {
+        assertEquals(KEYS_NAME, KEYS.name());
+    }
+
+    @Test
+    public void shouldMatchAutoDiscoverRealmsConfigName()
+    {
+        assertEquals(AUTO_DISCOVER_REALMS_NAME, AUTO_DISCOVER_REALMS.name());
+    }
+
```

Comment:
Move these test methods to `OAuthConfigurationTest` in same package as `OAuthConfiguration`, but in the test sources of course.